objcgen/classhelper.cs

```diff
@@ -25,6 +25,7 @@ public ClassHelper (SourceWriter headers, SourceWriter implementation)
 		public string Name { get; set; }
 		public string Namespace { get; set; }
 		public string ManagedName { get; set; }
+		public string OuterClass { get; set; }
```

Comment:
Can't `ManagedName` contain next nested name ? or does it break something else ?

Ideally we want to avoid checking `OuterClass` every time we use `ManagedName` (too easy to forget)